Shopping List/Shopping List/Controller/Table with shopping items/ItemsTableViewController.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  ItemsTableViewController.swift
+//  Shopping List
+//
+//  Created by Андрей Погосский on 3/2/20.
+//  Copyright © 2020 Андрей Погосский. All rights reserved.
+//
+
+import UIKit
+import os.log
+
+class ItemsTableViewController: UITableViewController {
```

Comment:
Please extend `ItemsTableViewController ` to conform to `UITableViewController` protocol explicitly.

```swift
extension ItemsTableViewController: UITableViewController { ... }
```

Reply:
Done.